yellowbrick/features/projection.py

```diff
@@ -0,0 +1,303 @@
+        the visualization as defined in other Visualizers.
+    """
+    
+    def __init__(self, ax=None, features=None, classes=None, color=None,
+             colormap=palettes.DEFAULT_SEQUENCE, target_type="auto", projection=2,
+             alpha=0.75, **kwargs):
+
+        super(ProjectionVisualizer, self).__init__(ax=ax, features=features, 
+                                                     classes=classes, color=color,
+                                                     colormap=colormap, 
+                                                     target_type=target_type, **kwargs)
+        # Convert string to integer
+        if isinstance(projection, str):
+            if projection in {'2D', '2d'}:
+                projection = 2
+            if projection in {'3D', '3d'}:
+                projection = 3
+        if projection not in {2, 3}:
+            raise YellowbrickValueError("Projection dimensions must be either 2 or 3")
+        self.projection = projection
+
+        if self.ax.name!='3d' and self.projection == 3:
+                warnings.warn("Uses third feature space as size. Pass 3d axes.", 
+                              YellowbrickWarning)
+
+        self.alpha = alpha
+        self._cax = None
+
+    @property
+    def cax(self):
+        """
+        The axes of the colorbar, right of the scatterplot.
+        """
+        if self._cax is None:
+            raise AttributeError(
+                "This visualizer does not have an axes for colorbar"
+            )
+        
+        return self._cax       
+        
+    @property
+    def ax(self):
+        """
+        Overloads the axes property from base class. If no axes is specified then
+        creates an axes for users. A 3d axes is created for 3 dimensional plots.
+        """
+        if not hasattr(self, "_ax") or self._ax is None:
+            if self.projection==3:
+                fig = plt.gcf()
+                self._ax = fig.add_subplot(111, projection='3d')
+            else:
+                self._ax = plt.gca()
+        return self._ax
+
+    @ax.setter
+    def ax(self, ax):
+        self._ax = ax
+
+    def _layout(self):
+        """
+        Creates the layout for colorbar when target type is continuous.
+        The colorbar is added to the right of the scatterplot.
+        """
+        # Ensure matplotlib version compatibility
+        if make_axes_locatable is None:
+            raise YellowbrickValueError((
+                "Colorbar requires matplotlib 2.0.2 or greater "
+                "please upgrade matplotlib"
+            ))
+
+        if self._cax is not None or self._target_color_type != TargetType.CONTINUOUS:
+            self.ax
+            return
+        # Create the new axes for the colorbar
+        divider = make_axes_locatable(self.ax)
+        self._cax = divider.append_axes("right", size="5%", pad=0.3)
+        self._cax.set_yticks([])
+        self._cax.set_xticks([])
```

Comment:
Why are we taking the ticks off the colorbar axes?

Reply:
I was getting ticks(extra) on the bottom and left of the axes. So I removed them.

Reply:
Ok, I was worried this would move the scale on the legend, but it looks like it's fine:

![Figure_1](https://user-images.githubusercontent.com/745966/61378062-84d80b00-a872-11e9-81fa-9ef0c6716752.png)

I guess this is because we are manually creating the colorbar in `finalize()`; let's just make sure this remains true as we move to the other visualizers.